scripts/Tools/compare_test_results

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+"""
+Analyze results from a test root area, comparing non-BFB
+changes. Purpose is, instead of re-running tests
+in compare mode, which is very slow, allow for very fast analsis of diffs.
+
+You may need to load modules for cprnc to work.
+"""
+
+from standard_script_setup import *
+
+from CIME.utils import expect
+from CIME.XML.machines import Machines
+from CIME.compare_test_results import compare_test_results
+
+import argparse, sys, os, doctest
+
+_MACHINE = Machines()
+
+###############################################################################
+def parse_command_line(args, description):
+###############################################################################
+    parser = argparse.ArgumentParser(
+usage="""\n%s [-n] [-r <TESTROOT>] [-b <BRANCH>] [-c <COMPILER>] [<TEST> <TEST> ...] [--verbose]
```

Comment:
This usage message is wrong: it's missing -t, and I don't see any documentation / meaning of the -n option